src/main.js

```diff
@@ -107,14 +112,20 @@ class NestedSort {
     wrapper.appendChild(list)
   }
 
+  getListTagName() {
+    return this.listInterface === HTMLOListElement ? 'ol' : 'ul'
```

Comment:
is it worth checking against `HTMLUListElement` just to be sure?
or are you in control of it being `ol` or `ul` so you 100% know if it's not OL then it's UL?

Reply:
I'd like to say that we are in control but I know that there would be some edge case like when we're dealing with a server rendered list which consists of both `ul` and `ol` lists.
I am going to create a new issue for this to implement a better error handling, maybe.